fcsparser/api.py

```diff
@@ -17,6 +17,7 @@
 import string
 import sys
 import warnings
+import re
```

Comment:
FYI The imports should be alphabetically sorted (PEP-8) -- i'll fix this up after the merge

Reply:
Oh, good to know.